### ISSUE TITLE
How can I compile modellauncher?

### ISSUE THREAD

**Issue by dlebauer** [PecanProject/pecan#613]
In attempting to solve #566, I tried to compile [modellauncher](https://github.com/PecanProject/pecan/tree/master/utils/modellauncher) on roger-login.ncsa.illinois.edu

### On Roger

When I try:
```sh
cd pecan/utils/modellauncher/
make
```

I get

> mpicc     modellauncher.c   -o modellauncher
make: mpicc: Command not found

If I load mpich:
 
```sh
module load mpich/3.1.4
make 
```

I get (the same as `gcc modellauncher.c`):

> modellauncher.c:4:21: error: mpi/mpi.h: No such file or directory
modellauncher.c: In function ‘main’:
modellauncher.c:65: error: ‘MPI_COMM_WORLD’ undeclared (first use in this function)
modellauncher.c:65: error: (Each undeclared identifier is reported only once
modellauncher.c:65: error: for each function it appears in.)
modellauncher.c:83: error: ‘MPI_INT’ undeclared (first use in this function)
modellauncher.c:83: error: ‘MPI_ANY_SOURCE’ undeclared (first use in this function)

The [Roger user guide](https://wiki.ncsa.illinois.edu/display/ROGER/ROGER+User+Guide) provides more information on using MPI - and there are apparently three different compilers.

### On Ubuntu 14.04 

```sh
sudo apt-get install libcr-dev mpich2
make
```

>mpicc     modellauncher.c   -o modellauncher
modellauncher.c: In function ‘main’:
modellauncher.c:83:4: error: too few arguments to function ‘MPI_Recv’
    MPI_Recv(buffer, 1, MPI_INT, MPI_ANY_SOURCE, 123, MPI_COMM_WORLD);
    ^
In file included from modellauncher.c:4:0:
/usr/include/mpi/mpi.h:1350:20: note: declared here
 OMPI_DECLSPEC  int MPI_Recv(void *buf, int count, MPI_Datatype datatype, int source,
                    ^
make: *** [modellauncher] Error 1

**TODO**

need to add / update documentation for configuration settings

**Comment by dlebauer** [comment]
@taolin1 @haohuuiuc let's look at this tomorrow. @robkooper can you join in the afternoon?

**Comment by robkooper** [comment]
think so, check my calendar.

**Comment by dlebauer** [comment]
fixed by #614
